VERSION

```diff
@@ -0,0 +1 @@
+1.11.9
```

Comment:
I really don't like the idea of two different version files committed to the repo.
One source of truth would be best